zane-test.fpcore

```diff
@@ -0,0 +1,4 @@
+(FPCore (x eps)
```

Comment:
Don't commit these. Again, please look over your diff before you commit & push!

Reply:
Sounds good. I left this in figuring I could remove it before merging into main. 

Is there a way to do local git ignore? Maybe the answer is not using `git add --all` as much as I do.  I had this and a `/test` dir that I was better about removing but slipped up this one and just ran with it.

Reply:
Don't do `git add -a`. Your work product is a diff, it's crazy not to actually inspect it. It'd be like not reading your code! If it's helpful, you can make a directory, then put a `.gitignore` file inside that directory with the contents `*`, and then git will ignore the whole directory. Probably call it `zane` in case we ever want to use `test`...

Reply:
Well my work flow currently is `git diff` then `git add --all`. I slipped on the `zane-test.fpcore` when I switch work flows over break.

Thanks for the tip about `/zane` I will remember that.

Reply:
Consider using `git add --patch` instead.